hangups/conversation.py

```diff
@@ -477,6 +485,87 @@ async def send_message(self, segments, image_file=None, image_id=None,
                 logger.warning('Failed to send message: {}'.format(e))
                 raise
 
+    @staticmethod
+    def _wrap_user_id(id_):
+        """Convert a user ID into a hangouts_pb2.InviteeID object."""
+        if isinstance(id_, hangouts_pb2.InviteeID):
+            return id_
+        elif isinstance(id_, user.UserID):
+            return hangouts_pb2.InviteeID(gaia_id=id_.gaia_id)
+        else:
+            return hangouts_pb2.InviteeID(gaia_id=id_)
+
+    @staticmethod
+    def _wrap_participant_id(id_):
+        """Convert a user ID into a hangouts_pb2.ParticipantId object."""
+        if isinstance(id_, hangouts_pb2.ParticipantId):
+            return id_
+        elif isinstance(id_, (user.UserID, hangouts_pb2.InviteeID)):
+            return hangouts_pb2.ParticipantId(gaia_id=id_.gaia_id)
+        else:
+            return hangouts_pb2.ParticipantId(gaia_id=id_)
+
+    async def add_users(self, user_ids):
+        """Add one or more users to this conversation.
+
+        Args:
+            user_ids: List of :class:`.InviteeID` of the new users.
```

Comment:
The docstrings for the two methods should be updated to reflect the three argument types that we're accepting in each.